packages/std/src/storage.rs

```diff
@@ -101,7 +101,7 @@ fn range_bounds(start: Option<&[u8]>, end: Option<&[u8]>) -> impl RangeBounds<Ve
 type BTreeMapPairRef<'a, T = Vec<u8>> = (&'a Vec<u8>, &'a T);
 
 #[cfg(feature = "iterator")]
-fn clone_item<T: Clone>(item_ref: BTreeMapPairRef<T>) -> Pair<T> {
+fn clone_item<T: Clone>(item_ref: BTreeMapPairRef<T>) -> Pair<Vec<u8>, T> {
```

Comment:
I see this as a cleaner approach than using <V, K> types, but also more breaking. (Switching the order of arguments to be `(v, k)` would be MUCH more breaking.

My question is how much does this break contract code. This will affect storage-plus, but if we update that (one hour maybe), then do we have to also update all the contracts that use range? Or is this just implicit info used by the compiler.

Reply:
I think it's just an implicit / type alias thing. Did the change to `(K,V)` and it was pretty straightforward.

Reply:
Sounds good